langchain/src/chains/llm_chain.ts

```diff
@@ -1,282 +1,3 @@
-import {
-  BaseLanguageModel,
-  BaseLanguageModelInterface,
-  BaseLanguageModelInput,
-} from "@langchain/core/language_models/base";
-import type { ChainValues } from "@langchain/core/utils/types";
-import type { Generation } from "@langchain/core/outputs";
-import type { BaseMessage } from "@langchain/core/messages";
-import type { BasePromptValueInterface } from "@langchain/core/prompt_values";
-import { BasePromptTemplate } from "@langchain/core/prompts";
-import {
-  BaseLLMOutputParser,
-  BaseOutputParser,
-} from "@langchain/core/output_parsers";
-import {
-  CallbackManager,
-  BaseCallbackConfig,
-  CallbackManagerForChainRun,
-  Callbacks,
-} from "@langchain/core/callbacks/manager";
-import { Runnable, type RunnableInterface } from "@langchain/core/runnables";
-import { BaseChain, ChainInputs } from "./base.js";
-import { SerializedLLMChain } from "./serde.js";
-import { NoOpOutputParser } from "../output_parsers/noop.js";
+import { type LLMChainInput, LLMChain } from "@langchain/core/legacy/chains";
```

Comment:
This is under the namespace of langchain/chains/llm in python rather than llm_chain

Reply:
I think mismatch is ok, nobody should be importing from core directly anyway and the extra entrypoint is slightly burdensome

No problem to add if you feel strongly